ledger/apply/asset.go

```diff
@@ -102,7 +103,14 @@ func AssetConfig(cc transactions.AssetConfigTxnFields, header transactions.Heade
 			Index:   basics.CreatableIndex(newidx),
 		}
 
-		return balances.PutWithCreatable(header.Sender, record, created, nil)
+		// Save asset params and holding, and register new creatable
+		err = balances.PutWithCreatable(header.Sender, record, created, nil)
+		if err != nil {
+			return err
+		}
+
+		// register new holding
+		return balances.Allocate(header.Sender, basics.CreatableIndex(newidx), basics.AssetCreatable, false, basics.StateSchema{})
```

Comment:
maybe: pass the `created` to `Allocate`, instead of passing all the arguments ?

Reply:
I'll check if it suitable for apps as well